README.md

```diff
@@ -513,7 +529,15 @@ Slide Content
 **Function Usage**
 
 ```jsx
-import slidesMarkdown from "raw-loader!markdown.md";
+const slidesMarkdown = `
```

Comment:
@ebrillhart when I try to use the raw loader I get this message....

![screen shot 2018-09-12 at 8 56 51 am](https://user-images.githubusercontent.com/20192033/45514987-c9c77880-b75b-11e8-9121-a647b113ddc0.png)

After googling for a bit and [raw-loader!](https://webpack.js.org/) seems to be the old way to import files. I am not sure how to properly document this since it appears to be apart of the webpack configuration.

Reply:
IIRC we have a tension with how CRA makes assumptions about loaders and spectacle's varying use of raw-loader (sometimes we want raw markdown, sometimes we want it processed). If it works, maybe we can just disable the warning?

Reply:
It does work in for the demo. If disabling the warning will not hurt anything we should do it. In this way it will comply with the original docs.